File: 프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.cs
using System;

public class Solution {
    public int[] solution(string my_string) {
        int[] answer = new int[52];
        string alpabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
        
        for (int i=0; i<52; i++) {
            foreach(char ch in my_string) {
                if (ch == alpabet[i]) {
                    answer[i]++;
                }
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(string my_string) 
{
    vector<int> v(52,0);
    
    for(char c : my_string)
    {
        if(isupper(c))
        {
            v[c-65]++;
        }
        else if(islower(c))
        {
            v[c-71]++;
        }
    }
    
    return v;
}